### ISSUE TITLE
Action bar - menu items in grouped button can't be reached with keyboard

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#714]
### Expected behavior

Items in dropdown menu can be reached and "clicked" using a keyboard
### Actual behavior

In the item group dropdown, the button can be reached and opened via the keyboard. The menu can't be entered with the keyboard. This is only an issue with the item group which is persistently displayed/has more complicated content. The item group menu which displays responsively at small form factor works fine in demo example. 

Tested on Mac - Safari, Chrome
### Steps to reproduce
1. Tab to menu button
2. Open menu button
3. Try to move down menu using keyboard (this is steps hit an issue)
4. Hit enter on a menu item

**Comment by josh-channin** [comment]
Hey @Blackbaud-MattGregg, I was just looking into this. If you add the href attribute in the <a> tag of the  I believe you get the expected behavior (Tab and then arrows keys to navigate the submenu). http://plnkr.co/edit/JVcVVUZevoZQ5BI3jk84?p=preview
@Blackbaud-PatrickOFriel

**Comment by Blackbaud-MattGregg** [comment]
@Blackbaud-PatrickOFriel - in the plnkr the subactions were reachable via the keyboard. When the menu became collapsed into one button (i.e., small form factor), the buttons in it couldn't be reached.

**Comment by Blackbaud-PatrickOFriel** [comment]
@Blackbaud-MattGregg we're talking about the submenu not being reachable using the keyboard, correct?

**Comment by Blackbaud-MattGregg** [comment]
Correct, once it collapses down into one button with a menu.

**Comment by Blackbaud-PatrickOFriel** [comment]
In [v1.6.10](https://github.com/blackbaud/skyux/releases/tag/v1.6.10)
